hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -183,7 +183,7 @@ message OMRequest {
 
   optional RevokeS3SecretRequest            RevokeS3SecretRequest          = 93;
 
-  optional PurgePathsRequest                purgePathsRequest              = 94;
```

Comment:
Is there a reason for this rename ?

Reply:
PurgePathRequest is refactored into an internal message inside PurgeDirectoryRequest to include volumeId and bucketId